src/base/RunnersRegistry.h

```diff
@@ -39,6 +39,12 @@ class RegisteredRunner
 public:
     /* Related methods below are declared in their calling order */
 
+    /* Startup events */
+
+    /// Called right before parsing squid.conf.
+    /// Meant for initializing/preparing configuration parsing facilities.
+    virtual void bootstrapConfig() {}
+
     /* Configuration events */
```

Comment:
Please move the new configuration-related event to the `Configuration events` group. We may need a set of startup events unrelated to configuration as well, of course, but this PR is not about that.

Reply:
Done in c6b6dbd